lib/onebox/engine/bandcamp_onebox.rb

```diff
@@ -10,12 +10,12 @@ class BandCampOnebox
       def placeholder_html
         og = get_opengraph
         escaped_src = ::Onebox::Helpers.normalize_url_for_output(og[:image])
-        "<img src='#{escaped_src}' height='#{og[:video_height]}' #{Helpers.title_attr(og)}>"
+        "<img src='#{escaped_src}' height='#{og.video_height}' #{Helpers.title_attr(og)}>"
```

Comment:
Shouldn't you use the following instead?

```suggestion
        "<img src='#{og.image}' height='#{og.video_height}' #{og.title_attr}>"
```